packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -437,6 +438,9 @@ class CachedArtifacts implements Artifacts {
       case TargetPlatform.linux_arm64:
       case TargetPlatform.windows_x64:
         return _getDesktopArtifactPath(artifact, platform, mode);
+      // Artifacts for windows-arm64 are not yet available.
```

Comment:
Consider adding a TODO:

```suggestion
      // Artifacts for windows-arm64 are not yet available.
      // TODO: https://github.com/flutter/flutter/issues/62597
```